### DISCUSSION TITLE
Cannot access public repos of organization

### DISCUSSION THREAD

**Discussion by kwinkunks** [community/community#84983] [Mobile]
### Select Topic Area

Bug

### Body

I'm not certain this is a bug, but it seems like one.

I cannot access Equinor's GitHub repos in my iOS GitHub app. I am in the organization, but have not authenticated with SSO (see below).

The Equinor org has [a public repo called `ert`](https://github.com/equinor/ert). I would like to access it from the app. 

- If I search for repositories with `ert`, all I can see is https://github.com/Ensembles/ert, which is archived.
- Its README has a link to the new project. I click and get *This content doesn't exist or you don't have permission to view it*. It does exist, it's public, and I'm a maintainer.
- If I try to do single sign-on in the GitHub app (MS AD or whatever it is), I get "You can't get there from here" and the advice to use the Edge browser. Not Explorer at least.
- If I search for organizations with `Equinor` then all Equinor's various orgs show up, but clicking on Equinor or Equinor-Playground (the only legit/current orgs) gets me *User or organization Equinor was not found*, which it obviously is. The other orgs are all visible no problem (Ensembles, Equinor-test, statoil, equinor-brazil, webviz, etc).

What is going on? Is this some garbled setting in the organization? Is it something weird about my user? Or is it a bug in the app?

(I have no trouble in the web app, everything works as expected there. And I'm on the latest iOS.)

**Comment by Iron-Ham** [comment]
Hello @kwinkunks,

I believe that you've encountered a bug in how we're validating SAML authentication on our GraphQL API. It seems we're being overly restrictive with regards to visibility of data on the GraphQL API when we're retrieving information from mobile. I'll file an internal bug and get this prioritized – this seems super frustrating; as a member of organization, your access to public data shouldn't be more restricted than folks who are outside of your organization. 

This would normally be resolved if you could trigger your SSO – but it seems that's not possible in this case. I'd love to learn more about why that is: is that an organizational policy, an issue with your SSO provider, or something else?

**Reply by kwinkunks** [reply]
Thanks for the response; seems like you're onto something.

As for the SSO side of it: I don't know. I wonder if it's to do with how we control 'managed' and 'unmanaged' devices. Am investigating at work.

**Reply by cglong** [reply]
Just +1ing this as an Android user at a different organization that also authenticates using SAML.
